src/mars.d

```diff
@@ -624,7 +619,7 @@ Language changes listed by -transition=id:
   =import,10378  revert to single phase name lookup
   =tls           list all variables going into thread local storage
 ");
-                        return EXIT_FAILURE;
+                        exit(EXIT_SUCCESS);
```

Comment:
Then it should already be broken, as `-man`, `-version` and `-help` are using the same approach.